modules/java_api/CMakeLists.txt

```diff
@@ -27,9 +27,7 @@ add_jar(inference_engine_jar ${java_source}
         OUTPUT_NAME ${PROJECT_NAME} 
         OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 
-if(ENABLE_TESTS)
-    add_subdirectory(tests)
-endif()
+add_subdirectory(tests)
```

Comment:
That means users are not able to disable Java tests.

Reply:
On the other hand - they are not huge and does not require extra build dependencies and there is no reason to disable them.

Reply:
Yes, contrib tests are significantly less than main IE tests. If we need to disable contrib tests, we can add extra options in future.